pages/vi/vi-github-issues.md

```diff
@@ -32,11 +32,8 @@ Every time you start to work on an issue you need to create a branch to keep you
 
 Now when you make changes to files on your local machine, you are actually making changes to files in your new branch. If you switch back to master, the files on your local machine revert to how they were before you made changes, since you are no longer on the new branch created.
 
-Hint: The default branch in your repository is called master. Keeping the master branch clean of any changes is a good practice so if you make any changes you don't want, you may revert back.
 
-Now you can go and make the proposed changes to your local files.
-
-Note: It's important that you are on the new branch, make sure you are on the correct branch with `git checkout branch-name` before committing. Use `git branch` to make sure you are on the correct branch.
+Note: Use `git-branch` to see which branch you are in. Master is the default branch and has to be clean of all changes to revert back to. Make sure you are on right branch using `git checkout branch-name` while making proposed changes to your local files and while committing.
```

Comment:
"Master is the default branch and has to be clean of all changes to revert back to." this should be written a bit. Something like:
"Your Master branch is the base working branch. It needs to remain untouched just in case you need to revert some changes back to a working version."